zest/releaser/vcs.py

```diff
@@ -140,6 +150,14 @@ def get_pyproject_toml_version(self):
         # Might be None, but that is fine.
         return result.get("project", {}).get("version")
 
+    def get_pyproject_toml_name(self):
+        if not os.path.exists("pyproject.toml"):
+            return
+        with open("pyproject.toml", "rb") as myfile:
+            result = tomllib.load(myfile)
```

Comment:
I'm a bit surprised that a pyproject.toml file needs loading as a binary file? Is that the right way?

Reply:
As per the tomli readme:
https://github.com/hukkin/tomli#parse-a-toml-file
> The file must be opened in binary mode (with the `"rb"` flag). Binary mode will enforce decoding the file as UTF-8 with universal newlines disabled, both of which are required to correctly parse TOML.